.github/workflows/python.yaml

```diff
@@ -1,19 +1,18 @@
-name: Build
+name: Python Tests
 
 on:
   push:
-    branches: [ master, main ]
+    branches: [master, main]
   pull_request:
-    branches: [ master, main ]
-
+    branches: [master, main]
 
 jobs:
   test:
     name: Run Tests
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ "3.10" ]
+        python-version: ["3.10"]
```

Comment:
Now that we officially support 3.9
https://github.com/sign-language-processing/pose/blob/master/src/python/pyproject.toml#L16
Do you mind changing this to 3.9? (I think it should be the same number in both places)